docs/agent-api.asciidoc

```diff
@@ -787,3 +787,34 @@ This enables log correlation to APM traces with text-only loggers.
 "trace.id=abc123 span.id=abc123"
 ----
 // end::currentTraceIds[]
+
+[[apm-register-custom-metrics]]
+==== `apm.registerMetric(name, callback)`
+==== `apm.registerMetric(name, labels, callback)`
```

Comment:
```suggestion
==== `apm.registerMetric(name[, labels], callback)`
```

(Sorry, I thought I already made that suggestion - looks like it got lost somehow. What I suggest is consistent with other API docs, and having two titles breaks the build anyway.)